src/lib/utils/evm.utils.ts

```diff
@@ -0,0 +1,27 @@
+import { BalanceItem, BalanceNftData, NftData } from 'lib/apis/temple/endpoints/evm/api.interfaces';
+
+import { EVM_NATIVE_CURRENCY_ADDRESS, EvmTokenMetadata } from '../metadata/types';
+
+export const isPositiveTokenBalance = (data: BalanceItem): data is NonNullableField<BalanceItem, 'balance'> =>
+  Boolean(data.balance && data.balance !== '0');
```

Comment:
You can compare `balance` and `token_balance` to `string` without a previous type check